### ISSUE TITLE
Character Select: One controller may sometimes control two player slots

### ISSUE THREAD

**Issue by BenGoulet** [symbol24/ProjectJuice#154]
When the players play a match and then go back to the Main Menu, it will sometimes happen that when a player presses the A Button to activate a character slot, two character slots will be activated and both will be controlled by this one controller. If the player proceeds to the action phase the two Maimframes will spawn, but only one will be controlled by the player.

Note this has ever only been seen on Controller 2 and up.

No precise reproduction steps were yet found.

**Comment by symbol24** [comment]
yeah, this is more unity controller shenanigans, that to fix this, it may be to costly

**Comment by symbol24** [comment]
very possible WNF candidate

**Comment by monstervictor** [comment]
This was fixed. We found a "possible cause" while working on https://github.com/symbol24/ProjectJuice/issues/151

**Comment by BenGoulet** [comment]
We reproduced it in a Build.exe version.
Before finishing even one match, we returned to the main menu and the first player to press A opened two different controller slots.

**Comment by symbol24** [comment]
we didint get this during our long playtest
